ballerina/types.bal

```diff
@@ -123,5 +133,47 @@ public enum Protocol {
     TLS
 }
 
+# The stream iterator object that is used to iterate through the stream messages.
+#
+class StreamIterator {
```

Comment:
shall we mark this class as `isolated` ?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-mqtt/pull/9/commits/041d5b9016d02da09f39901fe4d4970544edfb25